packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -93,12 +94,15 @@ export default class Editor implements IEditor {
         const plugins: EditorPlugin[] = [];
         getObjectKeys(corePlugins).forEach(name => {
             if (name == '_placeholder') {
-                arrayPush(plugins, options.plugins);
+                if (options.plugins) {
+                    arrayPush(plugins, options.plugins);
+                }
             } else {
                 plugins.push(corePlugins[name]);
             }
         });
 
+        options.zoomScale = options.zoomScale ?? 1;

```

Comment:
Better not change value of the passed in object. We don't know if it will throw exception when set a value, or if it reused by the caller. So better save the value to a local variable, e.g. do it in line 106